### PR TITLE
Rm always truthy condition

### PR DESCRIPTION
**Pull request by tkali**

### CODE REVIEW

File: app/assets/javascripts/lib/core/ad_manager.js [hunk 1]
```diff
@@ -38,9 +38,7 @@ define([ "jquery", "lib/core/ad_sizes", "lib/core/ad_unit" ], function($, adSize
       collapseEmptyDivs: true,
       enableSingleRequest: false,
       afterEachAdLoaded: function($adunit) {
```
Comment (WunderBart) [inline app/assets/javascripts/lib/core/ad_manager.js]: Does this fire on successful load or any load?

Comment (tkali) [inline app/assets/javascripts/lib/core/ad_manager.js]: On successfull load.

Comment (WunderBart) [inline app/assets/javascripts/lib/core/ad_manager.js]: :shipit:

**Comment by robuye** [comment]
:sheep: it

**Pull request merged**